frontend/spa/src/components/ImageLabeling.vue

```diff
@@ -263,93 +308,6 @@ export default {
         contrast: function() {
             return parseFloat(this.contrast_slider_value) / 100.;
         },
-        isDisabled: function() {
-            switch (this.active_tool) {
-                case 'freehand':
-                    this.stateNew = false;
-                    this.stateAppend = false;
-                    this.stateErase = false;
-                    this.stateOverlap = false;
-                    this.stateNoOverlap = false;
-                    break;
-                case 'polygon':
-                    this.active_overlap_mode = 'overlap';
-                    this.stateNew = false;
-                    this.stateAppend = false;
-                    this.stateErase = false;
-                    this.stateOverlap = false;
-                    this.stateNoOverlap = true;
-                    break;
-                case 'rectangle':
-                    this.active_mode = 'new';
-                    this.active_overlap_mode = 'overlap';
-                    this.stateNew = false;
-                    this.stateAppend = true;
-                    this.stateErase = true;
-                    this.stateOverlap = false;
-                    this.stateNoOverlap = true;
-                    break;
-                case 'point':
-                    this.active_mode = 'new';
-                    this.active_overlap_mode = 'overlap';
-                    this.stateNew = false;
-                    this.stateAppend = true;
-                    this.stateErase = true;
-                    this.stateOverlap = false;
-                    this.stateNoOverlap = true;
-                    break;
-                case 'circle':
-                    this.active_mode = 'new';
-                    this.active_overlap_mode = 'overlap';
-                    this.stateNew = false;
-                    this.stateAppend = true;
-                    this.stateErase = true;
-                    this.stateOverlap = false;
-                    this.stateNoOverlap = true;
-                    break;
-                case 'select':
-                    this.stateNew = true;
-                    this.stateAppend = true;
-                    this.stateErase = true;
-                    this.stateOverlap = true;
-                    this.stateNoOverlap = true;
-                    break;
-            }
-        },
-        disableDrawTools: function() {
```

Comment:
By removing the disableDrawTools function, the functionality of selecting allowed tools for label tasks is also removed. Can we add this functionality back with the drop down list?

Reply:
Aah, so certain labeling tasks will only allow you to certain drawing tools?

Reply:
Yes